File: Código/Loja/src/dados/Funcionario.java
package dados;

public class Funcionario {
    private int id;
    private String nome;
    private String cpf;
    private String rg;
    private int idade;

    public Funcionario(String nome, String cpf, String rg) {
        this.nome = nome;
        this.cpf = cpf;
        this.rg = rg;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getCpf() {
        return cpf;
    }

    public void setCpf(String cpf) {
        this.cpf = cpf;
    }

    public String getRg() {
        return rg;
    }

    public void setRg(String rg) {
        this.rg = rg;
    }

    public int getIdade() {
        return idade;
    }

    public void setIdade(int idade) {
        this.idade = idade;
    }

    @Override
    public String toString() {
        return "Funcionario [nome=" + nome + ", cpf=" + cpf + ", rg=" + rg + ", idade=" + idade + "]";
    }
}

File: Código/Python/Loja/src/dados/Funcionario.py
class Funcionario:
    def __init__(self, nome, cpf, rg):
        self.id = None  
        self.nome = nome
        self.cpf = cpf
        self.rg = rg
        self.idade = None  

    def __str__(self):
        return f"Funcionario [nome={self.nome}, cpf={self.cpf}, rg={self.rg}, idade={self.idade}]"